stage0/run.go

```diff
@@ -379,6 +382,27 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 	return string(bytes), nil
 }
 
+func addResolvConf(cfg RunConfig, rootfs string) {
+	content := "# Generated by rkt\n\n"
+	if len(cfg.DNSSearch) > 0 {
+		content += fmt.Sprintf("search %s\n", strings.Join(cfg.DNSSearch, " "))
+	}
+	for _, server := range cfg.DNS {
+		if server == "" {
+			continue
+		}
+		content += "nameserver " + server + "\n"
+	}
+	if len(cfg.DNSOpt) > 0 {
+		content += fmt.Sprintf("options %s\n", strings.Join(cfg.DNSOpt, " "))
```

Comment:
Validate DNS options maybe? There are only a few of those, validating will give better UX with a constant overhead